test/conformance/ingress/util.go

```diff
@@ -61,6 +61,10 @@ func CreateService(t *testing.T, clients *test.Clients, portName string) (string
 			Labels: map[string]string{
 				"test-pod": name,
 			},
+			Annotations: map[string]string{
+				// This annotation is for probe with mTLS enabled.
+				"sidecar.istio.io/rewriteAppHTTPProbers": "true",
```

Comment:
cc @tcnghia 

I have been going out of my way to avoid precisely this sort of special casing for Contour (riddling our resources with annotations), and we should keep Istio to the same bar.

Reply:
I removed the annotation and used the config in configmap instead. I hope I did not misunderstand your comment...

Reply:
No code changes makes me real happy :)

Reply:
I see. Although I tried to consider the code changes, this is not Knative issue but Isito's as the tests just deploy k8s pod. It would be difficult to solve by Knative's code.

Additionally, this UX issue is discussed in Istio side as:

https://github.com/istio/istio/issues/10357
https://github.com/istio/istio/pull/17941

as per latest update, they will enable the option by default after Istio v1.5+.